static/js/hotkey.js

```diff
@@ -762,6 +763,10 @@ exports.process_hotkey = function (e, hotkey) {
     case 'compose_quote_reply': // > : respond to selected message with quote
         compose_actions.quote_and_reply({trigger: 'hotkey'});
         return true;
+    case 'edit_message':
+        var row = current_msg_list.get_row(msg.id);
+        message_edit.start(row);
+        return true;
```

Comment:
What happens if the message is not editable (based on `get_editability`?).  E.g. one sent by another sender?  I think nothing happening is probably OK-ish, but certainly we don't want an error or anything.

Reply:
A simple solution might be that it just opens the message as "View Source"?
In the default case most messages should be editable (due to community topic editing).

Reply:
That is actually what happens right now. The GIF above shows some examples for this.

Reply:
@timabbott I tested this out for all possible cases including the one you mentioned when nothing (message or topic) is editable. Nothing happens as expected.
One thing we could do here is that in the Keyboard shortcuts page, let this page have a description that says something like "Edit or Copy Source of selected message".

I am not sure about the wording but I hope you get my point. I can add this change as a part of this commit if you want (and we decide what text to add there).

Reply:
I think "Edit message" is good for the documentation. 
(Even though we do an action when nothing is editable.)